packages/react/src/components/Breadcrumbs/index.ts

```diff
@@ -0,0 +1,34 @@
+import { BreadcrumbsItem } from './BreadcrumbsItem';
+import { BreadcrumbsLink } from './BreadcrumbsLink';
+import { BreadcrumbsList } from './BreadcrumbsList';
+import { BreadcrumbsNav } from './BreadcrumbsNav';
+import { BreadcrumbsRoot } from './BreadcrumbsRoot';
+
+export type { BreadcrumbsRootProps } from './BreadcrumbsRoot';
+export type { BreadcrumbsNavProps } from './BreadcrumbsNav';
+export type { BreadcrumbsListProps } from './BreadcrumbsList';
+export type { BreadcrumbsItemProps } from './BreadcrumbsItem';
+export type { BreadcrumbsLinkProps } from './BreadcrumbsLink';
+
+const Breadcrumbs = {
+  Root: BreadcrumbsRoot,
+  Nav: BreadcrumbsNav,
+  List: BreadcrumbsList,
+  Item: BreadcrumbsItem,
+  Link: BreadcrumbsLink,
+};
```

Comment:
We should have a jsdoc with an example here, like our other components.
See `Accordion` for an example